modules/app-configuration/configuration-stores/version.json

```diff
@@ -2,7 +2,6 @@
   "$schema": "https://aka.ms/bicep-registry-module-version-file-schema#",
   "version": "0.3",
   "pathFilters": [
-    "./main.json",
-    "./metadata.json"
+    "./main.json"
```

Comment:
Please format like the others (this is being formatted differently on save as per .vscode settings)

Reply:
fixed